zazu/plugins/eslint_styler.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+"""ESLint plugin for zazu."""
+import zazu.styler
+zazu.util.lazy_import(locals(), [
+    'json',
+    'os',
+    'subprocess'
+])
+
+__author__ = "Patrick Moore"
+__copyright__ = "Copyright 2018"
+
+
+class ESLintStyler(zazu.styler.Styler):
+    """ESLint plugin for code styling."""
+
+    def style_string(self, string, filepath):
+        """Fix a string to be within style guidelines.
+
+        Args:
+            string (str): the string to style
+            filepath (str): the filepath of the file being styled
+
+        Returns:
+            Styled string.
+
+        """
+
+        node_modules_path = None
+        eslint = 'eslint'
+        cwd = os.path.normpath(os.getcwd())
+        dirname = os.path.normpath(os.path.dirname(filepath))
+        loop_count = 0
+
+        while True and loop_count < 100:
+            loop_count = loop_count + 1
+            maybe_eslint = os.path.join(dirname, 'node_modules/eslint/bin/eslint.js')
```

Comment:
Probably don't want to assume forward slash separators.
```
local_eslint = os.path.join(['node_modules', 'eslint', 'bin', 'eslint.js'])

maybe_eslint = os.path.join(dirname, local_eslint)
```

Reply:
✔️